Sources/TuistGenerator/Extensions/Xcodeproj+Extras.swift

```diff
@@ -1,12 +1,20 @@
 import Foundation
 import XcodeProj
+import TuistGraph
 
 extension PBXFileElement {
     public var nameOrPath: String {
         name ?? path ?? ""
     }
 }
 
+extension PBXBuildFile {
+    public func applyPlatformFilters(_ filters: [PlatformFilter]?) {
```

Comment:
looks empty and nil means the same righy? let's remove the ambiguity
```suggestion
    public func applyPlatformFilters(_ filters: [PlatformFilter]) {
```

Reply:
This is a good usability thing to assess.  Within Xcode's file format, if no filters are specified then the item is associated with all platforms the target deploys to.  Adding more complication, Xcode only shows filters for platforms the target deploys to, so im not sure what would happen if a platform is filtered for a target that doesnt deploy to it (probably need a lint rule on this)